.github/workflows/build_and_deploy_hold.yml

```diff
@@ -0,0 +1,116 @@
+name: Build and deploy with a hold state
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  call_build:
+    uses: ./.github/workflows/build.yml
+    with:
+      build_script: build-locales
+
+  call_unit_test:
+    uses: ./.github/workflows/unit_test.yml
+    needs: call_build
+
+  call_misc_tests:
+    uses: ./.github/workflows/miscellaneous_tests.yml
+
+  call_acceptance:
+    uses: ./.github/workflows/acceptance.yml
+    needs: call_build
+    secrets:
+      BROWSERSTACK_USERNAME: ${{ secrets.BROWSERSTACK_USERNAME }}
+      BROWSERSTACK_ACCESS_KEY: ${{ secrets.BROWSERSTACK_ACCESS_KEY }}
+
+  call_format_branch_name:
+    uses: ./.github/workflows/format_branch_name.yml
+
+  extract_versions:
+    runs-on: ubuntu-latest
+    outputs:
+      minor_version: ${{ steps.vars.outputs.minor_version }}
+      major_version: ${{ steps.vars.outputs.major_version }}
+    steps:
+      - name: extract major and minor substrings
+        id: vars
+        run: |
+          MINOR_VERSION="$(echo "${{ github.ref_name }}" | cut -d '.' -f 1,2)"  
+          echo $MINOR_VERSION
+          echo ::set-output name=minor_version::${MINOR_VERSION}
+          MAJOR_VERSION="$(echo "${{ github.ref_name }}" | cut -d '.' -f 1)"
+          echo $MAJOR_VERSION
+          echo ::set-output name=major_version::${MAJOR_VERSION}
+
+  should_deploy_major_version:
+    runs-on: ubuntu-latest
+    outputs:
+      deploy_major_version: ${{ steps.vars.outputs.deploy_major_version }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: check if the next minor version from current tag version exists
```

Comment:
nit: Can we update this to indicate when a deploy should happen? In this case, we should deploy the next major version when the next minor doesn't exist

Reply:
something like this? `allow for major version deployment if next minor version from current tag doesn't exist`

Reply:
sure, that works